core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractLibrary.java

```diff
@@ -131,7 +131,7 @@
   @JoinColumn(name = "lastModifier", nullable = false)
   private User lastModifier;
 
-  @Formula(value = "(SELECT MAX(lcl.changeTime) FROM LibraryChangeLog lcl WHERE lcl.libraryId = libraryId)")
+  @Formula(value = "(SELECT MAX(lmod.changeTime) FROM LibraryChangeLog lmod WHERE lmod.libraryId = libraryId)")
```

Comment:
I kept this line because when I tried the Sample pattern, the JDBC Library was unhappy, and when I tried without the `@Formula` annotation, the Hibernate LibraryAdditionalInfo was unhappy.